### PR TITLE
Feat/#729 vertical tab improvements

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

- color changes
- use ! icon instead of text
- uikit: move vertical and horizontal tabs to common tabs folder 

**Type of Change**

- [ ] Bug
- [x] Styling
- [ ] New Feature

**Checklist before requesting review:**

- [x] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import VerticalTabs from 'uikit/VerticalTabs';
+import VerticalTabs from 'uikit/Tabs/Vertical';
```
Comment (hlminh2000) [inline components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx]: I don't think this exists eh?

Comment (hlminh2000) [reply components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx]: also I don't think we really need to do this, it's easier to keep it as `VerticalTabs` so vscode picks up the name more easily for auto import :P

Comment (ciaranschutte) [reply components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx]: @hlminh2000 reverted

Comment (hlminh2000) [reply components/pages/submission-system/program-clinical-submission/FilesNavigator/index.tsx]: missed this import it seems :P

**Pull request merged**